src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,50 @@
+import { useState } from 'react';
+import styles from './Dropdown.module.css';
+import { useOutsideClick } from '../../hooks/useOutsideClick';
+
+export default function Dropdown({
+  list,
+  value,
+  onSelect,
+  placeholder
+}: {
+  list: string[];
+  value: string;
+  onSelect: (value: string) => void;
+  placeholder: string;
+}) {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const handleToggle = () => setIsOpen((prev) => !prev);
+
+  const ref = useOutsideClick<HTMLDivElement>(() => setIsOpen(false));
+
+  return (
+    <div className={styles.wrapper} ref={ref}>
+      <button
```

Comment:
```suggestion
      <Button
```
공용 버튼 컴포넌트를 만들어주신 것 같아서 그 컴포넌트를 사용하는 것도 좋을 것 같아요~

Reply:
공용컴포넌트에 이미 까만색 스타일링이 되어있어서 사용하지 않았어요! Button 컴포넌트의 이름을 바꾸는 게 좋을까요? CTAButton처럼?

Reply:
공용 Button 컴포넌트의 스타일이 유연하게 동작할 수 있게 variant나 클래스네임을 받을 수 있으면 좋을 것 같은데, 그런게 아니라면 메타가 말씀해주신 것처럼 CTAButton 같은걸 만들어도 좋을 것 같네요🙃